lazydocker.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: lazydocker
+  version: 0.24.1
+  epoch: 0
+  description: The lazier way to manage everything docker
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/jesseduffield/lazydocker
+      tag: v${{package.version}}
+      expected-commit: be051153525b018a46f71a2b2ed42cde39a1110c
+
+  - uses: go/build
+    with:
+      packages: .
+      ldflags: |
+        -s -w
```

Comment:
Do we need -w? I thought it is set by default 🤔

Reply:
good catch, copy paste mistake from lazygit, correcting it.